vuepress/docs/next/tutorials/devops/backing-up-and-restoring-your-environment.md

```diff
@@ -4,118 +4,96 @@ redirectFrom: /next/tutorials/customize-the-platform/app-engine/building-prepack
+```
+cd entando-de-app
+```
+
+3. (Optional) Checkout a branch for your desired Entando version. You can review <https://github.com/entando/entando-de-app/releases> to determine the correct tag to use. Note: if you don't perform this step, you'll be creating an Entando application based on the latest _entando-de-app_ code which may not yet be released. 
+```
+git checkout -b my-test v6.3.68-fix-1
+```
+
+4.  Move the _resources_ and _protected_ folders from your Entando backup into your
+    `entando-de-app` application under `src/main/webapp`. You should override any content that is
     already there.
 
-5.  Build a docker image from the core app replacing the value of the tag in the `-t`  with the tag you want to use for your image.
-
-       mvn clean package
-       docker build . -f Dockerfile.wildfly -t <YOUR-USER>/<YOUR-REPO-NAME>:<YOUR-VERSION>
-
-6.  Create a repository on your docker account to house your new
-    application
-
-7.  Push the Image to your Repository
-
-        docker push <YOUR-USER>/<YOUR-REPO-NAME>:<YOUR-VERSION>
-
-8. Now we need to generate a new application for deployment to
-    Kubernetes using the helm chart
-
-    -   If you have an output from helm from before you can re-use it
-        and just apply the changes to the config map. In this case skip to step 12.
-
-    -   Or you can re-run helm and change the output
-
-9. Re-run the helm command for your environment
-
-10. Open the output yaml file from the helm command in the text editor
-    of your choice
-
-    -   For example: `vi training-alpha.yaml`
-
-11. In that file look for the `ConfigMap`
-
-    -   The config map defines all of the images that are available as
-        part of the deployment. They aren’t all used concurrently.
-
-    -   You can also view this config map in `kubernetes/OpenShift`
-
-12. Now you need to update the deployment to use your customized Wildfly
-    image. Find `entando-de-app-wildfly` in the config map
-
-    -   Note that we are changing the de-app but you could create a
-        custom version of any of the included images
-
-13. Change the version to match the version you used for your image
-
-14. Change the "organization" to the name of your main image repository
-    organization and if you aren’t using docker.io (DockerHub) then
-    change the registry as well.
-
-15. Save the file
-
-16. Follow the deployment steps you went through when you originally
-    deployed your Entando application to your Kubernetes instance
-
-17. Once deployed go to the _App Builder_ in your app
-
-28. Click _Go To Homepage_ and you should see your restored application.
+5.  Build the application
+```
+mvn clean package
+```
+6.  Create a repository named _entando-de-app-wildfly_ in Docker for the new application. The Entando Operator will expect this name when performing the initial install.
+
+7. Create a Docker image for the application. You'll need to provide your account and version.
```

Comment:
your account information? name? credentials?

Reply:
user name. Or organization name technically but user is good enough.